.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Report a bug you detected using preview-generator
+title: "Bug:"
+labels: bug
+assignees: ""
+---
+## Description and expectations
+
+<!-- Explain: -->
+<!-- - the bug and what is going wrong. -->
+<!-- - what needs to be done to consider the bug solved. -->
+
+<!-- If you know which component may be affected, feel free to mention it. -->
+
+### How to reproduce
+
+<!-- Detail steps to reproduce the bug: -->
+
+
+### Version information
+
+<!-- Optional - describe the environment which reveals the bug: -->
```

Comment:
I don't think this should be optional. Weird behavior on Preview Generator are highly dependent of the environment, and I find myself asking people for more information when they are not given

Reply:
```suggestion
<!-- describe the environment which reveals the bug: -->
```